kobo/apps/stripe/models.py

```diff
@@ -156,34 +158,69 @@ def make_add_ons_from_existing_charges():
         Returns the number of PlanAddOns created.
         """
         created_count = 0
+        # TODO: This should filter out charges that are already matched to an add on
         for charge in Charge.objects.all().iterator(chunk_size=500):
             if PlanAddOn.create_or_update_one_time_add_on(charge):
                 created_count += 1
         return created_count
 
     @staticmethod
-    def increment_add_ons_for_user(user_id: int, add_on_type: UsageType, amount: int):
+    def get_organization_totals(
+        organization: 'Organization', usage_type: UsageType
+    ) -> (int, int):
+        """
+        Returns the total limit and the total remaining usage for a given org.
+        and usage type.
+        """
+        usage_mapped = USAGE_LIMIT_MAP[usage_type]
+        limit_key = f'{usage_mapped}_limit'
+        limit_field = f'limits_remaining__{limit_key}'
+        usage_field = f'usage_limits__{limit_key}'
+        totals = PlanAddOn.objects.filter(
+            organization__id=organization.id,
+            limits_remaining__has_key=limit_key,
+            usage_limits__has_key=limit_key,
+            charge__refunded=False,
+            charge__payment_intent__status=PaymentIntentStatus.succeeded,
+        ).aggregate(
+            total_usage_limit=Coalesce(
+                Sum(Cast(usage_field, output_field=IntegerField()) * F('quantity')),
+                0,
+                output_field=IntegerField(),
+            ),
+            total_remaining=Coalesce(
+                Sum(Cast(limit_field, output_field=IntegerField())),
+                0,
+            ),
+        )
+
+        return totals['total_usage_limit'], totals['total_remaining']
+
+    @staticmethod
+    def increment_add_ons_for_organization(
```

Comment:
`increment` is confusing. We are subtracting amount from AddOn limit. 
Let's use another word. 

- decrement_add_ons_for_organization
- deduct_add_ons_for_organization
- subtract_add_on_minutes_for_organization

Reply:
Let's change it everywhere we use the concept of `increment`ing an add-on.

Reply:
I like "deduct" more.